core-services/prow/02_config/openshift/cloud-provider-vsphere/_pluginconfig.yaml

```
@@ -0,0 +1,12 @@
+approve:
+- commandHelpLink: ""
+  repos:
+  - openshift/cloud-provider-vsphere
+  require_self_approval: false
```

Do we want want to change this?
```suggestion
  require_self_approval: true
```

---

Hm... Don't know. 

If we want, lets change it in other our repos as well. Might be done in separate PR, wdyt?